ui/src/app/components/direct-connection/direct-connection.component.ts

```diff
@@ -32,6 +33,11 @@ export class DirectConnectionComponent implements OnInit {
     { value: 'postgres', displayName: 'PostgreSQL' },
   ]
 
+  dialect = [
+    { value: 'google_standard_sql', displayName: 'Google Standard SQL Dialect' },
+    { value: 'postgresql', displayName: 'PostgreSQL SQL Dialect' },
+  ]
+
```

Comment:
Instead of declaring the `dialects` here, can we derive this from somewhere, like `app.constants.ts` maybe?
We should have only one source of truth for the dialects in the web app.

Reply:
Moved it to app.constants.ts